cmd/url.go

```diff
@@ -87,8 +89,24 @@ The created URL can be used to access the specified component from outside the O
 			os.Exit(1)
 		}
 
+		componentPorts, err := url.GetComponentServicePortNumbers(client, componentName, applicationName)
+		checkError(err, "unable to get component exposed ports")
+
+		var portFound bool
+
+		for _, port := range componentPorts {
+			if urlPort == port {
```

Comment:
Is flag corresponding to urlPort mandatory?
Please correct me if I am wrong as I might be missing something